notebooks/sparse/quickstart.ipynb

```diff
@@ -37,15 +36,12 @@
         "# Install the required packages.\n",
         "\n",
         "import os\n",
+        "!pip install torch==2.0.0+cpu torchvision==0.15.1+cpu torchaudio==2.0.1 --index-url https://download.pytorch.org/whl/cpu > /dev/null\n",
```

Comment:
Why do we need this?

Reply:
The default pytorch version is 2.0.1, which is incompatible with DGL sparse.

Reply:
I see, let's file a issue to remind us for removing this after next release.

Reply:
Then maybe move this 2 command together so that users can uncomment them and run both together.